test/avrora/codec/plain_test.exs

```diff
@@ -154,17 +165,26 @@ defmodule Avrora.Codec.PlainTest do
 
   defp payment_payload, do: %{"id" => "00000000-0000-0000-0000-000000000000", "amount" => 15.99}
 
+  defp null_payment_payload,
```

Comment:
Let's introduce another schema with a more logical null in it. Also, may I suggest to make null as a postfix, like `..._payload_with_null`